e2e-tests/cypress/e2e/pages/your-business/companies-house-search/companyDetails.js

```diff
@@ -11,15 +11,17 @@ const companyDetails = {
   yourBusinessSummaryList: () => cy.get('[data-cy="companies-house-summary-list'),
 
   tradingName: () => cy.get('[data-cy="trading-name'),
-  tradingNameYesRadio: () => cy.get('[data-cy="trading-name-yes'),
-  tradingNameNoRadio: () => cy.get('[data-cy="trading-name-no'),
+  tradingNameHeading: () => cy.get('[data-cy="trading-name-heading'),
 
   tradingAddress: () => cy.get('[data-cy="trading-address'),
-  tradingAddressYesRadio: () => cy.get('[data-cy="trading-address-yes'),
-  tradingAddressNoRadio: () => cy.get('[data-cy="trading-address-no'),
+  tradingAddressHeading: () => cy.get('[data-cy="trading-address-heading'),
 
   companyWebsiteHeading: () => cy.get('[data-cy="company-website-heading'),
   companyWebsite: () => cy.get('[data-cy="company-website'),
+
+  phoneNumberHeading: () => cy.get('[data-cy="company-phone-heading'),
+  phoneNumberHint: () => cy.get('[data-cy="company-phone-hint'),
+  phoneNumber: () => cy.get('[data-cy="company-phone'),
```

Comment:
should we prefix with `-input` if it's the input? Doing this in other places

Also is e.g `companyWebsiteHeading` and `phoneNumberHeading` a heading or label? If a label can we rename please

Reply:
I have renamed from heading to label.  For now I am leaving it without input as I believe most are the full component rather than the input.  Will modify as I go forward